ludwig/schema/metadata/configs/llm.yaml

```diff
@@ -119,3 +119,17 @@ adapter:
     adapter_layers:
       ui_display_name: Adapter Layers
       expected_impact: 3
+quantization:
+  _oneOf:
+    allOf:
+      ui_display_name: Quantization
+      expected_impact: 3
+    none:
+      ui_display_name: No Quantization
+      expected_impact: 3
+  _meta:
+    expected_impact: 3
+    ui_component_type: radio_string_combined
```

Comment:
this is the same component type as `base_model`. Just confirming that you want that?

Reply:
I think so? Was going off the `adapter` meta, which should work more or less the same as this.

Reply:
It's just a question of how you would like it to render. `adapter` and `base_model` render as a set of radio buttons where each button's label is the title of each option and when you select one the nested properties of that option are rendered below it.

Without `radio_string_combined`, you would see a dropdown in place of the radio buttons (because the dropdown is the default way a oneof choice is rendered)

Reply:
Gotcha, yeah my preference would be a radio button for Quantization / No Quantization, then a dropdown for the bits field if Quantization is selected. Is that what the current behavior would be?

Reply:
yup, that would be the behavior with `radio_string_combined`